packages/mendel-middleware/index.js

```diff
@@ -106,7 +107,17 @@ function MendelMiddleware(opts) {
         var pack = bpack({raw: true, hasExports: true});
         var decodedResults = trees.findTreeForHash(params.bundle, params.hash);
         if (!decodedResults || decodedResults.error) {
-            return notFound(res, decodedResults && decodedResults.error);
+            return throwBundleError(res, 404, decodedResults && decodedResults.error, params);
```

Comment:
This function don't need "throw" in the name because you don't `throw` on the function body. `bundleError` is enough.